### ISSUE TITLE
Not posting in a way detected as a CI user

### ISSUE THREAD

**Issue by tempusfrangit** [lbragstad/keystone-performance#10]
Make sure you post in the correct manner so that your results show up as a proper CI output (look at how nova's 3rd party CIs report).

**Comment by lbragstad** [comment]
Agreed, currently we just leave a comment on the review. You mean to make it post in the success/fail section of the current jenkins jobs?

**Comment by tempusfrangit** [comment]
Yep! So that "toggle CI" makes it appear/disappear.
